cfme/tests/configure/test_ntp_server.py

```diff
@@ -0,0 +1,85 @@
+from fixtures.pytest_store import store
+from utils.conf import cfme_data
+from utils.log import logger
+from utils.wait import wait_for
+
+
+def appliance_date():
+    status, msg = store.current_appliance.ssh_client().run_command("date --iso-8601=hours")
+    return datetime.strptime(msg.rsplit('-', 1)[0], '%Y-%m-%dT%H')
+
+
+def check_ntp_grep(clock):
+    status, msg = store.current_appliance.ssh_client().run_command(
+        "cat /etc/ntp.conf| grep {}".format(clock))
+    return not bool(status), msg
+
+
+def test_ntp_crud():
+    """ Insert, Update and Delete the NTP servers """
+    # set from yaml file
+    configuration.set_ntp_servers(*cfme_data['clock_servers'])
+    # Set from random values
+    configuration.set_ntp_servers(*(fauxfactory.gen_alphanumeric() for _ in range(3)))
+    # Deleting the ntp values
+    configuration.set_ntp_servers()
+
+
+def test_ntp_server_max_character():
+    configuration.set_ntp_servers(*(fauxfactory.gen_alphanumeric(255) for _ in range(3)))
+
+
+def test_ntp_conf_file_update_check():
+    # Adding the ntp server values
+    configuration.set_ntp_servers(*cfme_data['clock_servers'])
+    for clock in cfme_data['clock_servers']:
+        status, wait_time = wait_for(lambda: check_ntp_grep(clock),
+            fail_condition=False, num_sec=60, delay=5)
+        assert status[0] is True, "Clock value {} not update in /etc/ntp.conf file".format(clock)
+
+    # Unsetting the ntp server values
+    configuration.set_ntp_servers()
+    for clock in cfme_data['clock_servers']:
+        status, wait_time = wait_for(lambda: check_ntp_grep(clock),
+            fail_condition=True, num_sec=60, delay=5)
+        assert status[0] is False, "Found clock record '{}' in /etc/ntp.conf file".format(clock)
+
+
+def test_ntp_server_check():
+    orig_date = appliance_date()
+    # status, past_date = store.current_appliance.ssh_client().run_command(
+    #     "date --date='10 days ago' +%y%m%d")
+    past_date = orig_date - timedelta(days=10)
+    status, result = store.current_appliance.ssh_client().run_command(
+        "date --set=\"$(date '%{} %%H:%%M')\"".format(past_date))
+    new_date = appliance_date()
+    if new_date != orig_date:
+        logger.info("Successfully modified the date in the appliance")
+        # Configuring the ntp server and restarting the appliance
+        # checking if ntp property is available, adding if it is not available
+        configuration.set_ntp_servers()
+        yaml_config = store.current_appliance.get_yaml_config("vmdb")
+        ntp = yaml_config.get("ntp", None)
+        if not ntp:
+            yaml_config["ntp"] = {}
+        # adding the ntp interval to 1 minute and updating the configuration
+        yaml_config["ntp"]["interval"] = '1.minutes'
+        store.current_appliance.set_yaml_config("vmdb", yaml_config)
+        # restarting the evmserverd for NTP to work
+        status, msg = store.current_appliance.ssh_client().run_command('service evmserverd restart')
```

Comment:
https://github.com/RedHatQE/cfme_tests/blob/master/utils/appliance.py#L1160 - please see this here, the appliance already has a good well used method to restart the evmserverd process. We should use this, it's up to you whether you do it rudely or not ;)